src/lib/rnp.c

```diff
@@ -1453,12 +1486,10 @@ rnp_armor_stream(
         return RNP_ERROR_WRITE;
     }
 
-    result = is_armor ? rnp_armor_source(&src, &dst, (pgp_armored_msg_t) data_type) :
```

Comment:
Ok, so this one line was better, because it was one line. We don't need whole new function.